src/env.rs

```diff
@@ -77,6 +86,11 @@ pub struct Environment {
     source: Option<Map<String, String>>,
 }
 
+#[derive(Clone, Debug)]
+pub enum TranslationType {
+    Kebab, // Translate '_' to '-'
```

Comment:
Would it make sense to add more here?

Reply:
We can go even further, depending on [`convert_case` crate](https://docs.rs/convert_case/latest/convert_case/enum.Case.html) and pass a `convert_case::Case` value here instead (eliminating `TranslationType`). If doing this, should we add a feature flag with optional dependency?

Reply:
That sounds good, IMO. Yes, a feature flag would be nice!